packages/amplication-code-gen-types/src/plugin-events-params.ts

```diff
@@ -93,3 +96,28 @@ export interface CreateServerPackageJsonParams extends EventParams {
 export interface CreateServerAppModuleParams extends EventParams {
   modulesFiles: Module[];
 }
+
+export interface CreateEntityResolverParams extends EventParams {
+  templatePath: string;
+  entityName: string;
+  entityServiceModule: string;
+  serviceId: namedTypes.Identifier;
+  resolverBaseId: namedTypes.Identifier;
+  mapping: ResolverMapping;
+}
+
+export interface CreateEntityResolverBaseParams extends EventParams {
+  templateBasePath: string;
+  entityName: string;
+  entityType: string;
+  entityServiceModule: string;
+  entity: Entity;
+  entityDTO: NamedClassDeclaration;
+  serviceId: namedTypes.Identifier;
+  resolverBaseId: namedTypes.Identifier;
+  createArgs: NamedClassDeclaration | undefined;
+  updateArgs: NamedClassDeclaration | undefined;
+  createMutationId: namedTypes.Identifier;
+  updateMutationId: namedTypes.Identifier;
+  mapping: ResolverMapping;
```

Comment:
same as above about the path and the mapping

Reply:
see my above comment